README.md

```diff
@@ -129,31 +128,32 @@ RMT is replacing some functionality of [SMT](https://github.com/SUSE/smt). Follo
 
 | Feature/Tech      | SMT           | RMT           |
 |-------------------|---------------|---------------|
-|Available on SLES11|yes|no|
-|Available on SLES12|yes|no|
-|Available on SLES15|no|yes|
-|Sync products data from SCC|yes|yes|
-|Mirror RPMs from repositories|yes|yes|
-|Selective mirroring(which products to mirror)|yes|yes|
-|Serve RPMs via http|yes|yes|
-|Registration of SLE 15 systems|yes|yes|
-|Registration of SLE 12 systems|yes|yes|
-|Registration of SLE 11 systems|yes|no|
-|Migration support SLE 12 > 15|yes|yes|
-|Staging repositories|yes|no<sup>[1](#staging)</sup>|
-|Air gap sync/mirroring for secure environments|yes|yes|
-|NTLM Proxy support|yes|yes|
-|Custom repositories|yes|yes|
-|YaST installation wizard|yes|yes|
-|YaST management wizard|yes|no|
-|Client management|yes|no|
-|RedHat support (Extended Support)|yes|no<sup>[2](#res)</sup>|
-|Files deduplication|yes|yes|
-|Data transfer from SMT to RMT|-|yes|
-|Transfer registration data to SCC|yes|no|
-|Reporting|yes|no|
-|Custom TLS certificates for web-server|yes|yes|
-|Web-server|Apache2|Nginx|
+|Available on SLES11|:white_check_mark:|:x:|
```

Comment:
note to self: check if available for Leap 42 and friends

Reply:
rmt-server itself is available on 12, but not the yast module. So it involves some manual setup to get it running.

Reply:
There's no documentation on the manual setup (on how to generate the SSL certs and such), though.

Reply:
Customer only can get RMT on SLES12 via non-supported channels, so it is effectively not available from my point of view. You can install it, for sure, but with manual configuration and SSL setup.